### DISCUSSION TITLE
[BUG] GitHub.com profile URLs going crazy

### DISCUSSION THREAD

**Discussion by gonzalo-cordova-pou** [community/community#37538] [Other Feature Feedback, Questions, & Ideas]
When in my profile I go to "followers" or "following" and I click other user, I am redirected to the other user URL but I still see my profile pic, username, description, details... (left panel) but the URL, "Overview" and repos are the ones of the other user.

See it yourself:

https://user-images.githubusercontent.com/71346949/198347732-67da7952-63ee-4521-94aa-13fb503247c7.mp4

**Comment by radar-blip** [comment]
Seems to be happening a lot today. If you refresh the page it will return to normal.

Update: Seems this is fixed now. 👍🏻
